src/main/java/org/prebid/server/bidder/yieldlab/YieldlabBidder.java

```diff
@@ -486,7 +492,8 @@ private ObjectNode resolveExtParameter(YieldlabDigitalServicesActResponse dsa) {
         try {
             dsaNode = mapper.mapper().convertValue(dsa, JsonNode.class);
         } catch (IllegalArgumentException e) {
-            throw new PreBidException("Failed to serialize DSA object", e);
+            logger.error("Failed to serialize DSA object for adslot {}", yieldlabResponse.getId(), e);
```

Comment:
I believe this error should be returned in the response, not just logged

Reply:
This would require a big refactoring of the existing implementation as there is no support for returning valid bids along with failed bids. Yieldlab did not write the existing implementation, and we feel it is beyond the scope of this ticket to be refactoring the existing implementation. Our goal is simply to add DSA support since this is a legal requirement. We are fine with not reporting failures.